Update the FullTextSearch docs to provide an example of their optionality (#25)
block/elasticgraph: config/site/src/query-api/filtering/full-text-search.md

{% highlight graphql %}
{{ site.data.music_queries.filtering.PhraseSearch }}
{% endhighlight %}

### Bypassing matchesPhrase and matchesQuery

> In order to ignore a `matchesPhrase` or `matchesQuery` filter, you can supply `null` to the `MatchesQueryFilterInput` parameter, as such:

Reviewer: ```suggestion
In order to make a `matchesPhrase` or `matchesQuery` filter optional, you can supply `null` to the `MatchesQueryFilterInput` parameter, like this:
```

We're going to be moving away from the language of "ignoring a filter" with the changes @acerbusace is working on since we very much have to _not_ ignore such a filter when it shows up inside an `anyOf`.  I think speaking of optional filters avoids that misunderstanding.

Author: Fixed, thanks for letting me know. I've readjusted the paragraph to match the phrasing you provided here. Let me know if further updates are required.
